bin/browserify-ftw.js

```diff
@@ -71,9 +74,7 @@ try {
 }
 
 try {
-  options = require(fullConfig);
-  // Poor man's validation
-  if (!options.quote || !options.style) throw new Error(fullConfig + 'is not a valid refactor config');
```

Comment:
It now always reads from the included one. If `quote` or `style` is missing from the users config, the default ones are used.
If the default ones are messed with, I say it should just fail spectacularly sometime later